packages/byods/src/byods/index.ts

```diff
@@ -37,9 +42,12 @@ export default class BYODS {
    * @example
    * const sdk = new BYODS({ clientId: 'your-client-id', clientSecret: 'your-client-secret' });
    */
-  constructor({clientId, clientSecret}: SDKConfig) {
+  constructor({clientId, clientSecret}: SDKConfig, config?: BYODSConfig) {
```

Comment:
Do you think it would be better to add logger config in SDKConfig rather than creating a new config for it,
```suggestion
  constructor({clientId, clientSecret, logger}: SDKConfig) {
```